docs/index.rst

```diff
@@ -56,10 +56,15 @@ variety of platforms.
 
     cloud/emr
 
+.. toctree::
+    :caption: Demo notebooks
+    :maxdepth: 2
+
+    notebooks/mango-python-alignment-executed.ipynb
+    notebooks/mango-python-coverage-executed.ipynb
 
 
-* :ref:`genindex`
```

Comment:
why is genindex removed?

Reply:
It appeared to have no useful content currently and was causing a weird linebreak